docs/user_guide/getting_started_with_moonstone.rst

```diff
@@ -141,5 +141,62 @@ Normalization
 Analysis
 """"""""
 
-Visualization
-"""""""""""""
+Plot
+"""""
+
+At any point of your work on your data, Moonstone provides standard plots to visualize your data and metadata.
+
+.. Note::
+    More details about available operations can be found in the API documentation about :ref:`api_plot`.
+
+
+How does it work?
+'''''''''''''''''
+
+To plot your data, you need to instantiate a `PlotStatsData` object with a pandas dataframe generated with moonstone (at any steps)
+
+.. code-block:: python
+
+    from moonstone.plot import PlotStatsData
+
+    # instantiation
+    plot_instance = PlotStatsData(df)
+
+    # call one (or more) plotting method(s)
+    plot_instance.YourFavoritePlot()
```

Comment:
careful here you are calling methods I think that usually are named with lower cases separated by underscores

Reply:
You're right!